src/components/encoding-pane/property-editorSchema.ts

```diff
@@ -0,0 +1,79 @@
+import * as vlSchema from 'vega-lite/build/vega-lite-schema.json';
+
+export const AXIS_TITLE_SCHEMA = {
+  "type": "object",
+  "title": "Axis",
+  "properties": {
+    "axisTitle": {
+      "type": "string",
+      "title": "Title"
+    },
+  },
+};
+
+export const STACK_SCHEMA = {
+  "type": "object",
+  "title": "Stack",
+  "properties": {
+    "stackSelect": {
+      "title": "Offset",
+      "enum": (vlSchema as any).definitions.StackOffset.enum,
+      "type": "string"
+    }
+  }
+};
+
+export const AXIS_ORIENT_SCHEMA = {
+  "type": "object",
+  "title": "Axis",
+  "properties": {
+    "orient": {
```

Comment:
Why isn't this combined in the same schema as `axisTitle`?

Why is it called "orient", not "axisOrient"?

Reply:
This answer the first question, but not 

> Why is it called "orient", not "axisOrient?

(Tip: use `>` to highlight a question u answer when reply a comment with multiple questions.)

Reply:
Ah thanks for the tip, did not know that :)

> Why is it called "orient", not "axisOrient"

Good question. The new schema convention will have the properties key be `prop_nestedProp` (or do you prefer just `propNestedProp`

Reply:
`prop_nestedProp` is way better (simpler to generate, easy to split)

Reply:
`propNestedProp` is more JS style though, but l still think `_` is better